### DISCUSSION TITLE
Quais os três melhores livros que você já leu?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#288] [Discussão]
POR FAVOR, só responda os três livros, com o link para mais informações

Tem alguns tópicos que envolvem livros no forum https://github.com/frontendbr/forum/issues/253, https://github.com/frontendbr/forum/issues/55 porém neste tópico, seu objetivo é apenas listar três. **Levando em consideração o tema do forum**.

Então:

**Comment by paulinhoerry** [comment]
[Javascript: The definitive guide](http://shop.oreilly.com/product/9780596805531.do)
[Javascript patterns](http://shop.oreilly.com/product/9780596806767.do)
[Head First HTML with CSS](http://headfirstlabs.com/books/hfhtml/)

**Comment by felipeuntill** [comment]
[jQuery Enlightenment](http://jqueryenlightenment.com/)
[Beginning SOLID Principles and Design Patterns for ASP.NET Developers](https://www.amazon.com/Beginning-Principles-Patterns-ASP-NET-Developers/dp/1484218477)
[Book of CSS3](https://www.nostarch.com/css3.htm)

**Comment by vitor-mendrone-deel** [comment]
[Secrets of the JavaScript Ninja](https://www.manning.com/books/secrets-of-the-javascript-ninja)
[Javascript Application Design](https://www.manning.com/books/javascript-application-design)
[Don't make me Think](https://en.wikipedia.org/wiki/Don%27t_Make_Me_Think)

**Comment by EmanoelLopes** [comment]
[Eloquent JavaScript ](http://eloquentjavascript.net/)
[JavaScript Patterns](http://shop.oreilly.com/product/9780596806767.do)
[JavaScript: The good parts](http://shop.oreilly.com/product/9780596517748.do)

**Comment by leticia** [comment]
[Eloquent JavaScript](http://eloquentjavascript.net/contents.html)
[If Hemingway Wrote JavaScript](http://anguscroll.com/hemingway/)
[The Mythical Man-Month: Essays on Software Engineering](https://en.wikipedia.org/wiki/The_Mythical_Man-Month)

**Comment by simaodeveloper** [comment]
[Segredos do Ninja JavaScript](http://novatec.com.br/livros/ninja-javascript/)
[Princípios de Orientação a Objetos em JavaScript](http://novatec.com.br/livros/orientacaoobjetosjavascript/)
[JavaScript de Alto Desempenho](http://novatec.com.br/livros/javascriptdesemp/)

**Comment by vitorabner** [comment]
[Pro Git](https://git-scm.com/book/en/v2)
[O andar do bêbado](http://www.saraiva.com.br/o-andar-do-bebado-como-o-acaso-determina-nossas-vidas-2658080.html)
[Practical Object-Oriented Design in Ruby]()

**Comment by willianjusten** [comment]
Sei lá, é meio complicado selecionar os 3 melhores, porque acho que cada um tem uma forma diferente de escrita, alvo e material. Mas talvez 2 dos mais memoráveis para mim são:
- [The Pragmatic Programmer: From Journeyman to Master](https://www.amazon.com.br/Pragmatic-Programmer-Journeyman-Master/dp/020161622X)
- [Clean Code: A Handbook of Agile Software Craftsmanship](https://www.amazon.com.br/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882)

**Comment by vitor-mendrone-deel** [comment]
@willianjusten tu conseguiu ler o Pragmatic Programmer? To a 6 meses lendo a primeira página hahahahah 

Clean Code é uma aula da primeira a ultima página, recomendo fortemente também, esqueci de colocar ele na minha lista!

**Comment by willianjusten** [comment]
Pow @Mendrone, já li o Pragmatic 3x, acho que foi o livro que mais abriu minha mente! Tenho ele sempre comigo <3
